tests/cypress/e2e/main/dex-markets.cy.ts

```diff
@@ -174,4 +167,29 @@ describe('DEX Pools', () => {
       )
     })
   })
+
+  /**
+   * Clicks on the given filter chip, opening the drawer on mobile if needed.
+   */
+  function clickFilterChip(chip: string, isMobile = breakpoint === 'mobile') {
```

Comment:
This isn't changed, just moved lower down. It doesn't use `withFilterChips` because in this case the filters close automatically